.github/workflows/coverage.meta

```diff
@@ -15,8 +15,8 @@
                 "-DBUILD_DOCUMENTATION=ON",
                 "-DBUILD_TESTS=ON",
                 "-DCMAKE_BUILD_TYPE=Debug",
-                "-DCOMPILE_EXAMPLES=ON",
-                "-DCMAKE_CXX_FLAGS='-Werror'",
+                "-DCMAKE_C_FLAGS='--coverage'",
```

Comment:
This has conflicts with the recently merged #178.

We should leave the original version of `test.meta`, since it is being used in the `flaky-test` job